### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DylanWelzel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,140 +1,38 @@
...
+
+## Overall Structure
+### Back End
+The back end was built with [Flask](https://flask.palletsprojects.com/en/2.0.x/), and it utilizes a postgreSQL database.
+
+#### Database Diagram
+![Database Diagram](https://github.com/Felmallakh/connect-team/blob/main/Database%20Schema.png)
+
+### Front End
+The front end was built with React.
```
Comment (kvh8899) [inline README.md:23]: We also used redux as a state manager so it is not just React by itself.

File: README.md [hunk 2]
```diff
@@ -1,140 +1,38 @@
...
+* log into an existing account
+* log out of an account
+* create/edit/delete and view workspaces
+* create/edit/delete and view channels inside of workspaces
+* LIVE chat for each channel
+* able to search for specific users
+
+## Overall Structure
+### Back End
+The back end was built with [Flask](https://flask.palletsprojects.com/en/2.0.x/), and it utilizes a postgreSQL database.
```
Comment (kvh8899) [inline README.md:17]: Capitalize PostgreSQL

File: README.md [hunk 3]
```diff
@@ -1,140 +1,38 @@
...
+
+#### Database Diagram
```
Comment (kvh8899) [inline README.md:19]: I dunno if the diagram is necessary here.

**Review by kvh8899 — CHANGES_REQUESTED** [review]

**Comment by kvh8899** [comment]
Some screenshots of the pages would be nice here. You don't have to put the splash page on here yet because I am going to try and work on it this week.

File: README.md (hunk 3 above)
Comment (DylanWelzel) [reply README.md]: Yeah I couldn't remember if it was a requirement or not for the readme even thought it's in the wiki.

**Pull request merged**